core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1525,6 +1525,7 @@ class PlaintextConsumerTest extends BaseConsumerTest {
   }
 
   @Test
+  @Disabled
```

Comment:
Add a comment about why it's disabled?  It's not a flake; this test case now doesn't work by design, right?